wagtail/admin/templates/wagtailadmin/shared/side_panels/includes/status/privacy.html

```diff
@@ -20,14 +20,19 @@
             {{ block.super }}
         {% endwith %}
     </div>
+    {% trans 'Change privacy' as set_privacy_title %}
+    {% trans 'Choose who can view the live version of this page' as set_privacy_message %}
+    {% trans 'Privacy changes apply to all children of this page too' as privacy_message_text %}
```

Comment:
```suggestion
    {% trans 'Privacy changes apply to all children of this page too.' as privacy_message_text %}
```

Let’s use the exact same text as before so people don’t have to re-translate this just for the dot.